zerver/lib/digest.py

```diff
@@ -369,6 +369,9 @@ def bulk_get_digest_context(
             hot_topic.teaser_data(user, stream_id_map) for hot_topic in hot_topics
         ]
 
+        context["total_hot_topic_conversations"] = sum(
+            hot_topic.teaser_data(user, stream_id_map)["count"] for hot_topic in hot_topics
```

Comment:
This is used as a number of messages, not a number of conversations, so the variable name is poor.

But I also think the value is misleading for the strings you have. This is the total number of messages in just the "hot topic" conversations, not the total number of new messages in that organization that the user can see.

I think it probably makes sense for the digest code to check earlier whether content can appear in email notifications, and instead do a `single `Message/UserMessage` query to get this count and then render the template, rather than doing all this `hot_topics` calculation that we won't even use.